images/komunikacja_mikroserwisow.dia

```diff
@@ -0,0 +1,3067 @@
+        <dia:boolean val="false"/>
+      </dia:attribute>
+      <dia:attribute name="flip_vertical">
+        <dia:boolean val="false"/>
+      </dia:attribute>
+      <dia:attribute name="subscale">
+        <dia:real val="1"/>
+      </dia:attribute>
+    </dia:object>
+    <dia:object type="Flowchart - Box" version="0" id="O25">
+      <dia:attribute name="obj_pos">
+        <dia:point val="97.165,-11.7804"/>
+      </dia:attribute>
+      <dia:attribute name="obj_bb">
+        <dia:rectangle val="97.115,-11.8304;109.473,-8.73482"/>
+      </dia:attribute>
+      <dia:attribute name="elem_corner">
+        <dia:point val="97.165,-11.7804"/>
+      </dia:attribute>
+      <dia:attribute name="elem_width">
+        <dia:real val="12.2575"/>
+      </dia:attribute>
+      <dia:attribute name="elem_height">
+        <dia:real val="2.9955844122715689"/>
+      </dia:attribute>
+      <dia:attribute name="show_background">
+        <dia:boolean val="true"/>
+      </dia:attribute>
+      <dia:attribute name="padding">
+        <dia:real val="0.5"/>
+      </dia:attribute>
+      <dia:attribute name="text">
+        <dia:composite type="text">
+          <dia:attribute name="string">
+            <dia:string>#Weryfikator tożsamości login/hasło#</dia:string>
+          </dia:attribute>
+          <dia:attribute name="font">
+            <dia:font family="sans" style="0" name="Helvetica"/>
+          </dia:attribute>
+          <dia:attribute name="height">
+            <dia:real val="0.80000000000000004"/>
+          </dia:attribute>
+          <dia:attribute name="pos">
+            <dia:point val="103.294,-10.0876"/>
+          </dia:attribute>
+          <dia:attribute name="color">
+            <dia:color val="#000000"/>
+          </dia:attribute>
+          <dia:attribute name="alignment">
+            <dia:enum val="1"/>
+          </dia:attribute>
+        </dia:composite>
+      </dia:attribute>
+    </dia:object>
+    <dia:object type="Flowchart - Box" version="0" id="O26">
+      <dia:attribute name="obj_pos">
+        <dia:point val="97.2065,-7.93272"/>
+      </dia:attribute>
+      <dia:attribute name="obj_bb">
+        <dia:rectangle val="97.1565,-7.98272;109.489,-5.38211"/>
+      </dia:attribute>
+      <dia:attribute name="elem_corner">
+        <dia:point val="97.2065,-7.93272"/>
+      </dia:attribute>
+      <dia:attribute name="elem_width">
+        <dia:real val="12.232947314527252"/>
+      </dia:attribute>
+      <dia:attribute name="elem_height">
+        <dia:real val="2.500609665440984"/>
+      </dia:attribute>
+      <dia:attribute name="show_background">
+        <dia:boolean val="true"/>
+      </dia:attribute>
+      <dia:attribute name="padding">
+        <dia:real val="0.5"/>
+      </dia:attribute>
+      <dia:attribute name="text">
+        <dia:composite type="text">
+          <dia:attribute name="string">
+            <dia:string>#Weryfikator tożsamości mID#</dia:string>
```

Comment:
mID to system identyfikacji, nie uwierzytelniania. Dlaczego zatem jest przypisany do menedżera sesji, obok „weryfikatora tożsamości login/hasło”? Czy „weryfikator tożsamości” oznacza narzędzie służące do uwierzytelniania?